### DISCUSSION TITLE
CP-SAT issue

### DISCUSSION THREAD

**Discussion by michaeldouble07** [google/or-tools#3129] [CP-SAT questions] [answered]
I am having trouble with this `import com.google.ortools.sat.LinearExprBuilder;` it seems like this does not exist, and I used the example from google.  Is there a different library?  I am using java, with maven.
Link to example used: [link: https://developers.google.com/optimization/assignment/assignment_groups](https://developers.google.com/optimization/assignment/assignment_groups)

**Comment by lperron** [comment, marked as answer]
This is from the master branch.
The last stable release does not contain this code.

Sorry for the confusion as the dev site picks the latest branch.

We should release 9.3 soon (this week or next week).
